packages/vue/package.json

```
@@ -2,7 +2,7 @@
   "name": "@carbon/charts-vue",
   "version": "0.11.8",
   "description": "Carbon charting components for Vue",
-  "main": "index.js",
+  "main": "dist/charts-vue.umd.min.js",
```

This should just be 
```suggestion
  "main": "charts-vue.umd.min.js",
```

Since we deploy the contents of the dist directory now